traffic_ops/bin/traffic_ops_ort.pl

```diff
@@ -1623,9 +1623,7 @@ sub check_lwp_response_content_length {
 	my $url           = $lwp_response->request->uri;
 
 	if ( !defined($lwp_response->header('Content-Length')) ) {
-		( $log_level >> $panic_level ) && print $log_level_str . " $url did not return a Content-Length header!\n";
-		exit;
-		return 1;
+		return 0; # Content-Length MAY be omitted per HTTP/1.1 RFC 7230, and in fact MUST NOT be included with a 'Transfer-Encoding: Chunked' header, which MUST be accepted by clients.
```

Comment:
Ah, `Content-MD5` was removed from HTTP/1.1 in https://tools.ietf.org/html/rfc7231#appendix-B . It's still legal to use, though, or we could use a custom `X-Content-MD5`.